cmd/eval.go

```diff
@@ -196,6 +197,13 @@ Set the output format with the --format flag.
 	--format=values    : output line separated JSON arrays containing expression values
 	--format=bindings  : output line separated JSON objects containing variable bindings
 	--format=pretty    : output query results in a human-readable format
+
+Schema
+------
+
+The -s/--schema flag provides a single JSON Schema used to validate references to the input document.
+
+	$ opa eval --data policy.rego --input input.json --schema input-schema.json
```

Comment:
nit: I'm inclined to rename this to `--input-schema` so that we don't run into naming issues in the future when we want to support loading of multiple schemas.

Reply:
@tsandall do you mean that in the future, we'll rename this flag (to upload schemas to be associated with something other than input)?

Reply:
Also, @tsandall based on our plan for the next upcoming PR (containing support for a. uploading multiple schemas via file(s) and directory path(s) and b. using rule based annotations with those in-memory schemas), our plan was to **re-use** this `opa eval --schema` argument. But the description for --schema in PR #2 would look something like this:

```--schema string                      set schema file(s) or directory path(s). This flag can be repeated.```

Given this next change ^^^, do you think we should leave --schema as-is, in this PR?

Reply:
If we leave the flag as-is then what's the behaviour going to be in the next PR?

* if a single file path is passed, the schema applies to `input` in whole.
* if a directory path is passed, the schemas do not necessarily apply to `input` in whole; something else is needed to associate schemas with `input` or subpaths under `input` or `data`.

I suppose that's a reasonable default. In most cases, people will want to apply schema to `input` so the flag does that in the simplest way possible. However, it can be overloaded to support more sophisticated use cases as well.

OK, I'm fine with it as-is :)